config/configuration.go

```diff
@@ -121,6 +136,22 @@ func SetProjectRoot(args []string) error {
 	return setCurrentProjectEnvVariable()
 }
 
+// UniqueID gets the unique installation ID.
+func UniqueID() string {
+	configDir, err := common.GetConfigurationDir()
+	if err != nil {
+		APILog.Warning("Unable to read config dir, %s", err)
+		return ""
+	}
+	idFile := filepath.Join(configDir, ".gauge_id")
```

Comment:
The file is already in `.gauge` folder. So, the filename can just be `.id` or `id`. Thoughts?